cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -285,8 +286,11 @@ var CMS = window.CMS || {};
 
                 var bounds = this.options.plugin_restriction;
                 var type = clipboardPlugin.data('settings').plugin_type;
+                var parent_bounds = clipboardPlugin.data('settings').plugin_parent_restriction;
+                var currentPluginType = this.options.plugin_type;
 
-                if (bounds.length && $.inArray(type, bounds) === -1) {
+                if ((bounds.length && $.inArray(type, bounds) === -1) ||
+                    (parent_bounds.length && $.inArray(currentPluginType, parent_bounds) === -1)) {
```

Comment:
this is what is supposed to fix clipboard, but parent bounds is always an empty array